src/components/RegisterModal/RegisterModal.js

```diff
@@ -1,12 +1,46 @@
 import { useState } from "react";
 import PopupWithForm from "../PopupWithForm/PopupWithForm";
-const RegisterModal = ({ handleModalClose, handleLoginModal }) => {
+const RegisterModal = ({
+  handleModalClose,
+  handleLoginModal,
+  userSignUpAccount,
+  emailNotFoundError,
+}) => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
   const [userName, setUserName] = useState("");
+  const [emailError, setEmailError] = useState("");
+  const [passwordError, setPasswordError] = useState("");
+  const [userNameError, setUserNameError] = useState("");
+
+  const ValidateForm = () => {
```

Comment:
Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)